pkg/check/soc/soc.go

```diff
@@ -13,22 +13,46 @@ import (
 	"github.com/ethersphere/bee/pkg/crypto"
 	"github.com/ethersphere/bee/pkg/soc"
 	"github.com/ethersphere/beekeeper/pkg/bee"
-	"github.com/prometheus/client_golang/prometheus/push"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
 )
 
-// Options represents SOC check options
+// Options represents check options
 type Options struct {
-	RequestTimeout time.Duration
 	PostageAmount  int64
-	PostageWait    time.Duration
 	PostageDepth   uint64
+	PostageWait    time.Duration
+	RequestTimeout time.Duration
+}
+
+// NewDefaultOptions returns new default options
+func NewDefaultOptions() Options {
+	return Options{
+		PostageAmount:  1,
+		PostageDepth:   16,
```

Comment:
maybe we can inject a PostageConfig struct here and everywhere else so that we do not repeat ourselves.

Reply:
there are global options and we can move it there